### PR TITLE
add count in meta field

### PR DESCRIPTION
**Pull request by southeo**
Include total count of hits for specimens and annotations by a given creator. Value is stored in meta field. 

Jira: 

https://naturalis.atlassian.net/browse/DD-619?atlOrigin=eyJpIjoiYWE4NTc3NWU2NjI5NDFiYTg3MTI3ZDkyNGQ2ODk5NzAiLCJwIjoiaiJ9

https://naturalis.atlassian.net/browse/DD-620?atlOrigin=eyJpIjoiYjUxNDA1OGMxMmY1NGM5MDhjMjcxN2E1MDNhNjU1ODYiLCJwIjoiaiJ9

### CODE REVIEW

File: src/main/java/eu/dissco/backend/service/SpecimenService.java [hunk 1]
```diff
@@ -64,15 +64,16 @@ public class SpecimenService {
   private final AnnotationService annotationService;
   private final MongoRepository mongoRepository;
 
-  public JsonApiListResponseWrapper getSpecimen(int pageNumber, int pageSize, String path) {
-    var digitalSpecimenList = repository.getSpecimensLatest(pageNumber, pageSize);
-    return wrapListResponse(digitalSpecimenList, pageSize, pageNumber, path);
+  public JsonApiListResponseWrapper getSpecimen(int pageNumber, int pageSize, String path)
```
Comment (samleeflang) [inline src/main/java/eu/dissco/backend/service/SpecimenService.java:67]: Wondering, are the db methods still used?

File: .github/workflows/.trivyignore [hunk 2]
```diff
@@ -1,4 +1,10 @@
 # Date: June 16, 2023
 # Issue: snappy-java high issue, https://www.cvedetails.com/cve/CVE-2023-34453/
 # Solution: Spring needs to update its version
-CVE-2023-34455
\ No newline at end of file
+CVE-2023-34455
```
Comment (samleeflang) [inline .github/workflows/.trivyignore:4]: Is this still relevant or can we upgrade spring?

File: src/main/java/eu/dissco/backend/service/SpecimenService.java (hunk 1 above)
Comment (southeo) [reply src/main/java/eu/dissco/backend/service/SpecimenService.java:67]: Nope. Thought they were, but only in testing...methods (and tests) are removed

File: .github/workflows/.trivyignore (hunk 2 above)
Comment (southeo) [reply .github/workflows/.trivyignore]: We use latest version of spring-kafka, which uses 3.4 kafka by default, but according to [this](https://spring.io/projects/spring-kafka), kafka 3.5 should be compatible with spring. 

Manually upgraded kafka in the meantime, made a note in trivyignore.

**Pull request merged**